### PR TITLE
AppLab: New show/hide styling for toolbox header

### PR DESCRIPTION
**Pull request by islemaster**
Next step in applab style cleanup: Replacing the "(hide)" link in the toolbox header with a nice arrow icon.

circle-chevron-slide

I've also expanded the hit area for this function to include the entire toolbox header.

Here's the old version for comparison (also pre-slim-header):

out

I'm working from this mockup:

060115_codemode_flow2

The icon being used is [chevron-circle-right from Font Awesome](http://fortawesome.github.io/Font-Awesome/icon/chevron-circle-right/).  For the "hide" version I just mirror it horizontally in the stylesheet with `tranform: scale(-1, 1)` which works on our supported browsers [according to Can I Use...](http://caniuse.com/#feat=transforms2d)

Note about RTL support: I make a small effort to point the icon in the correct direction for the header collapse-expand motion in RTL, but it looks pretty broken right now because the toolbox is still on the left.  We're leaving it for now because applab doesn't officially support RTL yet, and it's easy to remove the bit of special-case style here if we decide to move the header back to the left.

### CODE REVIEW

File: apps/style/common.scss [hunk 1]
```diff
@@ -809,15 +828,43 @@ $aniGifPreviewHeight: 60px;
   -moz-user-select: none;
   -ms-user-select: none;
   user-select: none;
+
+  #show-toolbox-icon, #hide-toolbox-icon {
+    position: absolute;
```
Comment (islemaster) [inline apps/style/common.scss]: These get absolutely positioned so they don't interfere with centering of the header text.

File: apps/style/common.scss [hunk 2]
```diff
@@ -854,6 +895,20 @@ $aniGifPreviewHeight: 60px;
   div {
     margin-right: 0;
   }
+
+  #show-toolbox-icon, #hide-toolbox-icon {
```
Comment (islemaster) [inline apps/style/common.scss]: These are the RTL-specific styles.

**Comment by islemaster** [comment]
@cpirich suggested animating the arrow (rotating from one state to the other) which sounds awesome, but they're two separate elements at the moment and I'm not sure I should spend another hour getting that to work. :smile:

**Comment by breville** [comment]
looks good!

**Pull request merged**